robot-server/tests/integration/http_api/runs/test_pause_then_cancel.tavern.yaml

```
@@ -0,0 +1,90 @@
+test_name: Test a JSONv6 run can be paused and then cancelled.
+
+marks:
+  - usefixtures:
+      - run_server
+
+stages:
+  - name: Upload a JSONv6 protocol
+    request:
+      url: '{host:s}:{port:d}/protocols'
+      method: POST
+      files:
+        files: 'tests/integration/protocols/wait_then_home_v6.json'
+    response:
+      status_code: 201
+      save:
+        json:
+          protocol_id: data.id
+
+  - name: Create run from protocol
+    request:
+      url: '{host:s}:{port:d}/runs'
+      method: POST
+      json:
+        data:
+          protocolId: '{protocol_id}'
+    response:
+      status_code: 201
+      save:
+        json:
+          run_id: data.id
+
+  - name: Play the run
+    request:
+      url: '{host:s}:{port:d}/runs/{run_id}/actions'
+      method: POST
+      json:
+        data:
+          actionType: play
+    response:
+      status_code: 201
+
+  - name: Pause the run
+    request:
+      url: '{host:s}:{port:d}/runs/{run_id}/actions'
+      method: POST
+      json:
+        data:
+          actionType: pause
+    response:
+      status_code: 201
+
+  - name: Wait for the delay to complete
+    max_retries: 10
+    delay_after: 0.2
+    request:
+      url: '{host:s}:{port:d}/runs/{run_id}/commands'
+      method: GET
+    response:
+      status_code: 200
+      strict:
+        - json:off
+      json:
+        data:
+          - status: succeeded
+          - status: queued
```

What do two `status`es mean here?

---

These are the statuses of the protocol's two commands

---

I see. I think I'm being thrown off by the YAML syntax. No change needed, but if anyone else finds it confusing, I think a comment like this could be helpful.

```suggestion
          # Assert that there's a succeeded command followed by a queued command.
          - status: succeeded
          - status: queued
```